### PR TITLE
Add draft version of VIP-TokenRecovery

### PR DESCRIPTION
**Pull request by VetMaik**
This is a draft version of the VIP-TokenRecovery

### CODE REVIEW

**Comment by NecoSherry** [comment]
Hi there! Thank you for submitting your proposal. Feel free to make adjustments to your pull request based on the community discussion. Once your proposal is prepared, please request review from the operation team (@NecoSherry).

**Comment by VetMaik** [comment]
Based on feedback from Brettski, I made a start on the actual guides.
I also added two additional ideas I had that work very well with the guides.

1. With the release of the official vechain bridge, it would be great to be able to swap VET from different networks. Since other networks like BSC and Cronos have their own bridged VET, this would help our holders a lot. Instead of sending wrongly withdrawn VET back to the CEX, they can easily swap it to real VET from our official bridge.

2. I would like VeWorld to have custom derivation path support. Sync2 supports this since a few months and it is a game changer. It made my guides more easy to follow and a lot safer.

**Comment by VetMaik** [comment]
Hi @NecoSherry ! I had some discussion with Brettski on Discourse and added one of the guides in this pull request!
I wonder what you think of this.

**Comment by sebaseek** [comment]
Hi @VetMaik thanks for your contribution

just a chore text thing I saw in the following sentence: 

"Open Sync2 on your mobile device or computer and choose the import wallet option"

I would add quote symbol in the word import to express the desired CTA functionality

"Open Sync2 on your mobile device or computer and choose the 'import' wallet option"

Is there anything else you would like to add or we can proceed to review it completely

**Comment by VetMaik** [comment]
@sebaseek I added a second article for Binance Smart Chain and added your feedback as well.
It's now ready to review.

**Review by bacco1977 — APPROVED** [review]
ok

**Comment by bacco1977** [comment]
I completed my review thanks @VetMaik 
If the rest of the team has some further comments I have none

**Review by nwbrettski — APPROVED** [review]
LGTM.

**Comment by nwbrettski** [comment]
@NecoSherry can we progress to the next steps with this and close the VIP out. Thanks.

**Comment by VetMaik** [comment]
@NecoSherry I made the requested changes and relocated/renamed the files.

File: assets/vip-230/articles/Sent VET on the VechainThor network to BSC Address.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Did you sent your VET tokens to your Binance Smart Chain (BSC) address? This guide will help you retrieve them.
```
Comment (daithihearn) [inline assets/vip-230/articles/Sent VET on the VechainThor network to BSC Address.md:1]: Typo. Change *sent* to *send*

File: assets/vip-230/articles/Sent VET on the VechainThor network to Ethereum Address.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Did you sent your VET tokens to your Ethereum address? This guide will help you retrieve them.
```
Comment (daithihearn) [inline assets/vip-230/articles/Sent VET on the VechainThor network to Ethereum Address.md:1]: Typo. Change **sent** to **send**

**Review by daithihearn — CHANGES_REQUESTED** [review]

File: assets/vip-230/articles/Sent VET on the VechainThor network to Ethereum Address.md (hunk 2 above)
Comment (VetMaik) [reply assets/vip-230/articles/Sent VET on the VechainThor network to Ethereum Address.md:1]: Thanks again!

**Review by VetMaik — COMMENTED** [review]
Thanks for finding!

**Review by daithihearn — APPROVED** [review]

**Review by VetMaik — COMMENTED** [review]
Done

**Comment by libotony** [comment]
Hi @VetMaik, can you add links to the articles in file vip-230.md?

**Comment by Andredreyer1** [comment]
.

**Review by Andredreyer1 — COMMENTED** [review]
.

**Comment by Andredreyer1** [comment]
.

**Comment by Andredreyer1** [comment]
.

**Pull request merged**